### ISSUE TITLE
Improving bald and regular status bars

### ISSUE THREAD

**Issue by UriahShaulMandel** [UriahShaulMandel/BaldPhone#53]
Improving bald and regular status bars as mentioned by @DJCrashdummy :
> i also kind of like to get remembered of low energy (even while using an app) by highlighting the usual status bar in red... it would be nice if the "group-colour" of the "baldphone-statusbar" would also change, for people who never display the usual status bar and don't have a status LED.
> i'm still not sure if this should be an extra setting, or triggered automatically when the usual status bar is hidden. 🤔

**Comment by DJCrashdummy** [comment]
i'm not 100% sure if we are talking about the same enhancement... or this translation is a bit upside down: https://github.com/UriahShaulMandel/BaldPhone/commit/b17bd1b49f00506442fa157c8cbb64d8e33f4aa0#diff-01dafb3fd0217441330103cfc8300094R372
> It won\'t be shown if the system status bar is hidden.

then there is still no indication for low battery if you are just having baldphone on your display. :confused:

**Comment by UriahShaulMandel** [comment]
Full battery:
![image](https://user-images.githubusercontent.com/43012268/64913503-eec54180-d749-11e9-8bbe-4f3541e85593.png)
Medium battery:
![image](https://user-images.githubusercontent.com/43012268/64913510-056b9880-d74a-11e9-8d6b-ae6d438451ff.png)
Low battery:
![image](https://user-images.githubusercontent.com/43012268/64913521-23d19400-d74a-11e9-9cb9-7dad3a8836cc.png)
Low battery when status bar is on:
![image](https://user-images.githubusercontent.com/43012268/64913528-419ef900-d74a-11e9-8f70-fd273d2f9b07.png)

Did you mean for the hole BaldStatusBar to be red? I think it'll be a bit too red
Example
![image](https://user-images.githubusercontent.com/43012268/64913615-0f8e9680-d74c-11e9-94a8-e6fee61b7e06.png)
Of course, I can change it to be a bit better, but imo it's a mess...

**Comment by UriahShaulMandel** [comment]
Perhaps the Clock background can be red,  example for you to have an idea:
![image](https://user-images.githubusercontent.com/43012268/64913823-dbb57000-d74f-11e9-8b98-598efbc9c507.png)
(of course i'll make the corners circrled)

**Comment by UriahShaulMandel** [comment]
You can test out the changes I made this far here:
[BaldPhone_battery_and_notifications_test.zip](https://github.com/UriahShaulMandel/BaldPhone/files/3613171/BaldPhone_battery_and_notifications_test.zip)

**Comment by DJCrashdummy** [comment]
you are completely right! and the screenshots from your first post look good, except the last one... :nauseated_face: 

i haven't even thought about setting the android status bar to red via baldphone UI (which is a good idea) because i already have done so forever within the android UI.

basically the idea was about an indication while i'm in an "baldphone app" (like contacts, photos, alarms, etc.) and therefore colouring the baldphone "statusbar/locationbar/headerbar" or whatever you call it... instead of the baldphone group-/row-colour (green, grey, brown, etc.).

- - -
i guess a combination would be the best solution and the easiest to understand & handle for users:
- colouring the battery icon for full & low battery is good and should definitively stay.
IMHO this is sufficient at baldphones homescreen and colouring the whole baldphone statusbar looks really bad and also colouring the clock is a bit of overshooting. - in the worst case this could scare some users.
- just add one simple setting to *colour the "top bar" when the battery is low.*
and this sets the android status bar to red and also the baldphone header bar whenever the android status bar is not shown.

-> so there is just one easy setting without notes, ifs and whens... and no matter whether the android status bar is never, partly or always shown, there is always an indication of low battery without beeing overshooting, overwhelming or scaring.

**Comment by UriahShaulMandel** [comment]
Ah, only now I understood what you wanted :)
Working on it :)

**Comment by UriahShaulMandel** [comment]
After creating this, I think there is a chance users won't like this behavior. It also means registering another receiver for each activity, and reconstructing the BaldTitleBar.

**Comment by apkproz** [comment]
HI Bro, Share your app for free On APKPROZ store.
https://apkproz.com/promote-apps

**Comment by aha999** [comment]
I think it would be better if just the square space around the drained battery logo was highlighted red
